services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminExecutionTask.java

```diff
@@ -103,7 +103,7 @@ public class AdminExecutionTask implements Callable<Void> {
       boolean isParentController,
       AdminConsumptionStats stats,
       String regionName) {
-    this.LOGGER = LOGGER;
+    this.LOGGER = LogManager.getLogger(AdminExecutionTask.class);
```

Comment:
I think this is on purpose. Check caller: `this.LOGGER = LogManager.getLogger(consumerTaskId);`